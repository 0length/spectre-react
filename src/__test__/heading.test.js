import React from 'react'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'

import { Heading } from '../'

describe('Heading', () => {
  describe('Snapshots', () => {
    it('Should exist', () => {
      expect(Heading).toMatchSnapshot()
    })
  })
})
