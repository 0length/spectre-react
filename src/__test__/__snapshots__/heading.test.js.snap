// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading Snapshots Should exist 1`] = `[Function]`;
